### PR TITLE
[Cinder][Backup] Adding cinder backup-deployment feature

### PR DESCRIPTION
**Pull request by rajivmucheli**
Hi @fwiesel,

Please validate the files, i see cinder service up in staging, no errors in backup logs. But i get ERROR: Service cinder-backup could not be found. More details are noted in Jira CCM-352. Please advise how to fix this.
 
Regards,
Rajiv

### CODE REVIEW

File: openstack/cinder/templates/backup-deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            timeoutSeconds: 5
+          readinessProbe:
+            httpGet:
+              path: /healthcheck
+              port: cinder-backup
+            initialDelaySeconds: 15
+            timeoutSeconds: 5
+          volumeMounts:
+            - name: cinder-etc
+              mountPath: /tmp/cinder-backup.sh
```
Comment (fwiesel) [inline openstack/cinder/templates/backup-deployment.yaml]: Can you remove the script (also from the chart and the configmap)? It is not needed

File: openstack/cinder/templates/backup-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+            - name: cinder-etc
+              mountPath: /etc/cinder/cinder_audit_map.yaml
+              subPath: cinder_audit_map.yaml
+              readOnly: true
+      volumes:
+        - name: etccinder
+          emptyDir: {}
+        - name: cinder-etc
+          configMap:
+            name: cinder-etc
```
Comment (fwiesel) [inline openstack/cinder/templates/backup-deployment.yaml]: We need a configmap for the cinder-backup-<availability-zone>, containing the file "cinder-backup.conf", holding two config values

```
[DEFAULT]
host = cinder-backup-{{ az}}
storage_availability_zone = {{ region }}{{ az}}
```

Comment (fwiesel) [reply openstack/cinder/templates/backup-deployment.yaml]: That is, additionally to the cinder-etc configmap

**Comment by rajivmucheli** [comment]
i tried changing the values in backup.yaml by introducing values in secrets but it did not pick up, not sure if i going in the right direction

**Comment by rajivmucheli** [comment]
https://github.com/sapcc/helm-charts/pull/526

**Pull request closed without merging**